tools/maketests

```diff
@@ -237,18 +236,29 @@ sub stringifyDom {
   return [grep { /\S/ } split("\n", $string)]; }
 
 sub gen_xml {
-  my ($dir, $name) = @_;
+  my ($dir, $name, $is_test) = @_;
```

Comment:
`$is_test` doesn't really distinguish the cases, perhaps `$is_compare` or  `$for_comparison` or something like that?

Reply:
Sure thing, renamed :+1: